docs/reference/openapi-rules.md

```diff
@@ -846,3 +846,79 @@ schemas:
       - name
       - petType
 ```
+
+### oas3-server-variables
+
+This rule ensures that server variables defined in OpenAPI Specification 3 (OAS3) and 3.1 are valid, not unused, and result in a valid URL. Properly defining and using server variables is crucial for the accurate representation of API endpoints and preventing potential misconfigurations or security issues.
+
+**Recommended**: Yes
+
+**Bad Examples**
+
+1. **Missing definition for a URL variable**:
+
+```yaml
+servers:
+  - url: "<https://api>.{region}.example.com/v1"
```

Comment:
Why is there `< >` around `<https://api>`?

Reply:
@mnaumanali94?